test/unit/components/shepherd-footer.spec.js

```diff
@@ -58,10 +58,12 @@ describe('components/ShepherdFooter', () => {
     expect(cancelButton).toHaveAttribute('tabindex', '0');
     expect(cancelButton).toHaveClass('shepherd-button-secondary cancel-button shepherd-button');
     expect(cancelButton).toHaveTextContent('Exit');
+    expect(cancelButton).toHaveAttribute('aria-label', 'Exit');
```

Comment:
Could you please add a separate test for labels? We should test what happens when nothing is set for label, when label is set to null, undefined, string, number, etc and handle all the cases.

Reply:
Is that good enough for you?

Reply:
@lex111 perhaps I was not clear enough. I would like a separate test for labels please. Not adding an `expect` to the existing test, but a separate test to check label functionality. It should cover all the different cases I mentioned: what happens when nothing is set for label, when label is set to null, undefined, string, number, etc

Reply:
@rwwagner90 Sorry, I really misunderstood.
Need a separate `.spec` file for cancel button test? Or how best to do in this case?

Reply:
@lex111 no, just a separate test in the same file please

Reply:
@lex111 I can add the tests, if you would like

Reply:
@rwwagner90 sorry for the long answer.. yes, please add tests, I’m unlikely to do it soon.